.github/workflows/cli_core_e2e_test.yaml

```diff
@@ -30,10 +30,19 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-go-
 
+      - name: Setup dependencies
+        run: |
+          wget -O- https://carvel.dev/install.sh > install_carvel.sh
```

Comment:
nit: we have been seting up tools dependencies in hack/tools/bin (buildable with `make tools`). This gives us more control over the exact version of each tool used. We should make the paths to binaries configurable and preferably default to those locations.

Reply:
I have updated to use the `make tools`